book/09-git-and-other-scms/sections/import-bzr.asc

```diff
@@ -0,0 +1,154 @@
+==== Bazaar
+(((Bazaar)))(((Importing, from Bazaar)))
+
+Bazaar is a distributed source management content like Git.
+It is very easy to convert a Bazaar repository into a Git repository thanks to Bazaar's plugin system.
+To convert a Bazaar repository into a Git repository, you have first to import the bzr-fastimport plugin.
+Here is the whole procedure if you have not yet imported any plugin:
+[source,console]
+----
+$ mkdir --parents ~/.bazaar/plugins/bzr # creates the necessary folders for the plugins
+$ cd ~/.bazaar/plugins/bzr
+$ bzr branch lp:bzr-fastimport fastimport # imports the fastimport plugin
+$ cd fastimport
+$ sudo python setup.py install --record=files.txt
+----
+
+There you go!
+You've just installed the fastimport plugin.
+
+
+===== Case of a project with a single branch
+
+Now go in the directory that contains your Bazaar repository and initialize the Git repository:
+[source,console]
+----
+$ cd path/to/the/bzr/repository
+$ git init
+----
+
+Now, you can simply export your Bazaar repository and convert it into a Git repository using the following command:
+[source,console]
+----
+$ bzr fast-export --plain . | git fast-import
+----
+
+You may get the following error message:
+[source,console]
+----
+bzr: ERROR: Unable to import library "fastimport": bzr-fastimport requires the fastimport python module
+----
+
+This message is displayed because the fastimport python module that the `bzr fast-export` command needs is missing.
+Install it from your package manager if you are under GNU/Linux or download it at address https://pypi.python.org/pypi/fastimport/[] if you are under Windows.
+For example, with Fedora, you would do the following:
+
+[source,console]
+----
+$ sudo dnf install python-fastimport
+----
```

Comment:
This belongs in the section where we installed the plugin. Say something like

> To make sure this works, try this:
> ```
> $ [something that works if everything is installed correctly]
> ```
> If you get an error, you're probably missing a Python module. To correct this, do this:
> ```
> $ pip install fastimport
> ```

(Prefer a command that works almost anywhere the user could have Bazaar installed. Pip comes with most Python installations these days.)